netdata-installer.sh

```diff
@@ -1115,6 +1115,19 @@ function get_kernel_version() {
   printf "%03d%03d%03d" "${p[0]}" "${p[1]}" "${p[2]}"
 }
 
+function get_rh_version() {
+  if [ ! -f /etc/redhat-release ] ; then
+     echo "000000000"
```

Comment:
```suggestion
     printf "000000000"
```

Reply:
Thank you I changed the PR changing echo to printf.